__tests__/apps.test.js

```diff
@@ -195,3 +197,63 @@ describe("4. PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("5. GET /api/articles", () => {
+  test("status:200, returns all the articles if query is omitted", () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles).toHaveLength(12);
+        let articles = body.articles;
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("status:200, articles are sorted by date in descending order by default", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("status:200, articles are sorted by topic in descending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=topic")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("topic", { descending: true });
```

Comment:
this test should be looking for an array of only articles that have the topic you specify, for example - if the query is mitch then it should return an array of all the articles that have "mitch" as the topic and only those articles